prime-router/src/test/kotlin/azure/ReportFunctionTests.kt

```diff
@@ -71,6 +71,87 @@ class ReportFunctionTests {
+        "^^CMS&2.16.840.1.113883.3.249&ISO^^^^NPI||(123)456-7890^WPN^PH^^1^123^4567890|20210802||||||Winchester " +
+        "House|6789 Main St^^San Jose^CA^95126^^^^06085|(123)456-7890^WPN^PH^^1^123^4567890|6789 Main St^^San " +
+        "Jose^CA^95126\n" +
+        "OBR|1|1234d1d1-95fe-462c-8ac6-46728dba581c^Winchester House^05D2222542^ISO|" +
+        "1234d1d1-95fe-462c-8ac6-46728dba581c^Winchester House^05D2222542^ISO|94558-4^SARS-CoV-2 " +
+        "(COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay^LN^^^^2.68|||" +
+        "202108020000-0500|202108020000-0500||||||||1679892871^Doolittle^Doctor^^^^^^" +
+        "CMS&2.16.840.1.113883.3.249&ISO^^^^NPI|(123)456-7890^WPN^PH^^1^123^4567890|||||202108020000-0500|||F\n" +
+        "OBX|1|CWE|94558-4^SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid " +
+        "immunoassay^LN^^^^2.68||260415000^Not detected^SCT|||N^Normal (applies to non-numeric results)^" +
+        "HL70078^^^^2.7|||F|||202108020000-0500|05D2222542^ISO||10811877011290_DIT^10811877011290^99ELR^^^" +
+        "^2.68^^10811877011290_DIT||202108020000-0500||||Winchester House^^^^^ISO&2.16.840.1.113883.19.4.6&" +
+        "ISO^XX^^^05D2222542|6789 Main St^^San Jose^CA^95126^^^^06085\n" +
+        "OBX|2|CWE|95418-0^Whether patient is employed in a healthcare setting^LN^^^^2.69||N^No^HL70136||" +
+        "||||F|||202108020000-0500|05D2222542||||202108020000-0500||||Winchester House^^^^^" +
+        "ISO&2.16.840.1.113883.19.4.6&ISO^XX^^^05D2222542|6789 Main St^^San Jose^CA^95126-5285^^^^06085|||||QST\n" +
+        "OBX|3|CWE|95417-2^First test for condition of interest^LN^^^^2.69||N^No^HL70136||||" +
+        "||F|||202108020000-0500|05D2222542||||202108020000-0500||||Winchester House^^^^" +
+        "^ISO&2.16.840.1.113883.19.4.6&ISO^XX^^^05D2222542|6789 Main St^^San Jose^CA^95126-5285^^^^06085||" +
+        "|||QST\n" +
+        "OBX|4|CWE|95421-4^Resides in a congregate care setting^LN^^^^2.69||Y^Yes^HL70136||||||F|" +
+        "||202108020000-0500|05D2222542||||202108020000-0500||||Winchester House^^^^^ISO&2.16.840.1.113883.19.4.6&ISO" +
+        "^XX^^^05D2222542|6789 Main St^^San Jose^CA^95126-5285^^^^06085|||||QST\n" +
+        "OBX|5|CWE|95419-8^Has symptoms related to condition of interest^LN^^^^2.69||N^No^HL70136|||" +
+        "|||F|||202108020000-0500|05D2222542||||202108020000-0500||||Winchester House^^^^^ISO&" +
+        "2.16.840.1.113883.19.4.6&ISO^XX^^^05D2222542|6789 Main St^^San Jose^CA^95126-5285^^^^06085|||||QST\n" +
+        "SPM|1|1234d1d1-95fe-462c-8ac6-46728dba581c&&05D2222542&ISO^1234d1d1-95fe-462c-8ac6-46728dba581c&&" +
+        "05D2222542&ISO||445297001^Swab of internal nose^SCT^^^^2.67||||53342003^Internal nose structure" +
+        " (body structure)^SCT^^^^2020-09-01|||||||||202108020000-0500|20210802000006.0000-0500"
+
+    val hl7_5_separator = "MSH|^~\\&|CDC PRIME - Atlanta,^2.16.840.1.114222.4.1.237821^ISO|Winchester" +
```

Comment:
Is `|^~\\&|` supposed to include the `#`? Just checking this isn't a copy pasta mistake

Reply:
Good catch!  Yes, it is supposed to include the '#'. Fixing.